python/ray/rllib/README.rst

```diff
@@ -0,0 +1,66 @@
+RLLib: Ray's scalable reinforcement learning library
+====================================================
+
+Getting Started
+---------------
+
+You can run training with
+
+::
+
+    python train.py --env CartPole-v0 --alg PolicyGradient --s3-bucket s3://rllib
+
+The available algorithms are:
+
+-  ``PolicyGradient`` is a proximal variant of
+   `TRPO <https://arxiv.org/abs/1502.05477>`__.
+
+-  ``EvolutionStrategies`` is decribed in `this
+   paper <https://arxiv.org/abs/1703.03864>`__. Our implementation
+   borrows code from
+   `here <https://github.com/openai/evolution-strategies-starter>`__.
+
+-  ``DQN`` is an implementation of `Deep Q
+   Networks <https://www.cs.toronto.edu/~vmnih/docs/dqn.pdf>`__ based on
+   `OpenAI baselines <https://github.com/openai/baselines>`__.
+
+-  ``A3C`` is an implementation of
+   `A3C <https://arxiv.org/abs/1602.01783>`__ based on `the OpenAI
+   starter agent <https://github.com/openai/universe-starter-agent>`__.
+
+Storing logs in Amazon S3
```

Comment:
Since it looks like smart_open() supports multiple URI types, I would suggest renaming the flag to `--upload-dir`, and in the example, show how you can use S3 and Athena.

It also looks like if you don't specify the upload-dir then the result jsons are not written anywhere, should we by default always write it to the tmpdir as well?